.gitignore

```diff
@@ -12,3 +12,5 @@
 
 # Generated project artifacts
 **/dist
+
+__coverage__
```

Comment:
I think this is out of scope for this PR :smiley:

Reply:
yeah :smile: